### ISSUE TITLE
Jellyfin Integration Page - Depends on Cloud

### ISSUE THREAD

**Issue by gottsman** [koying/jellyfin_ha#4]
On the recently redesigned Home Assistant integration page, there is now detail about custom integrations and "connects to the cloud". The dashboard card identifies that this Jellyfin integration connects to the cloud... I don't believe this to be true. Perhaps it's a bug or further explanation is required in the README.

**Comment by koying** [comment]
Yeah, was wondering as well.
Mine happens to be in the cloud, so that's the reason, but the intent is maybe to mean *public* cloud.

**Comment by gottsman** [comment]
Right. Cloud connectivity is the reason I moved away from lifetime Plex. I don't think that Jellyfin requires the cloud.

**Comment by BelgarionNL** [comment]
nope jellyfin does not require cloud :)
